nuget.config

```diff
@@ -1,13 +1,25 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+    <add key="azure-sdk-for-net-dev" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
   </packageSources>
 
   <packageSourceMapping>
     <!-- key value for <packageSource> should match key values from <packageSources> element -->
+    <clear />
+    <packageSource key="dotnet-tools">
```

Comment:
We should be using Nuget.org for this now right?

Reply:
Yup, done.

Reply:
Removing.